// Copyright (c) 2008 Johan Euphrosine
// See LICENSE for details.

option py_generic_services = true;

message TestRequest {
  required string text = 1;
}

message TestResponse {
  required string text = 1;
}

service TestService {
  rpc TestMethod (TestRequest) returns (TestResponse);
}
